### ISSUE TITLE
intermediate network loss

### ISSUE THREAD

**Issue by oneTimePad** [oneTimePad/classification#10]
Sometimes a network has loss components that are not at the final network output. Examples of this are auxiliary logits. This could be achieved by allowing the user to output multiple network outputs in dict/list returned from the predict function of the FeatureExtractor.
